src/web/demo.rkt

```diff
@@ -602,7 +519,7 @@
   (define config
     `(init rand ,(get-seed) flags ,(*flags*) num-iters ,(*num-iterations*) points ,(*num-points*)
            timeout ,(*timeout*) output-dir ,(*demo-output*) reeval ,(*reeval-pts*) demo? ,(*demo?*)))
-  (thread-send *worker-thread* config)
+  (start-job-server config *demo?* *demo-output* )
```

Comment:
Willing to merge despite, but it would be nice if this init message lived in `server.rkt` instead of here. I think it should be doable; most of the fields are globals that you could read in `server.rkt`, and `*demo?*` and `*demo-output*` are passed in.